templates/scripts/devbase.sh.tpl

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# This script ensures we have a copy of devbase
+set -e
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+libDir="$DIR/../.bootstrap"
```

Comment:
Should we switch this to be `.devbase`?

Reply:
Yes, but we can't right now because I know for a fact that things have hardcoded that path 😢